patch/kernel/archive/rockchip64-6.3/dt/rk3399-orangepi-4-lts.dts

```diff
@@ -862,8 +862,6 @@
 
 &i2s0 {
 	rockchip,i2s-broken-burst-len;
-	assigned-clocks = <&cru SCLK_I2SOUT_SRC>;
-	assigned-clock-parents = <&cru SCLK_I2S0_8CH>;
 	resets = <&cru SRST_I2S0_8CH>, <&cru SRST_H_I2S0_8CH>;
 	reset-names = "reset-m", "reset-h";
```

Comment:
yes, it can go further, but it need more work to do. like jack detect, mainline jack detect for simple-card is not working, need to remove jack detection from dts otherwise headphone will not work.

Reply:
https://github.com/zhangn1985/linux-stable/commit/078dcc8e72c29a9a9507c8cb6c9ef3a7c269334e

https://github.com/zhangn1985/linux-stable/commit/c7442c70af453bd8000f34489d388a5e14cc3bc5